react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import ReactDOM from "react-dom"
+
+
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+
+
+function PokemonList(props){
+  const pokemon = props.name;
+  const listItems = pokedex.map((pokemon) =>
+  <li key={pokemon.toString()}>{ pokemon.name } </li>
```

Comment:
For the `key` prop you can just use the `number` property here: `pokemon.number`